packages/core/contracts/financial-templates/common/TokenFactoryExclusiveMinter.sol

```diff
@@ -0,0 +1,36 @@
+pragma solidity ^0.6.0;
+
+import "./SyntheticTokenExclusiveMinter.sol";
+import "../../common/interfaces/ExpandedIERC20ExclusiveMinter.sol";
+import "../../common/implementation/Lockable.sol";
+
+
+/**
+ * @title Factory for creating new mintable and burnable tokens.
+ */
+
+contract TokenFactoryExclusiveMinter is Lockable {
```

Comment:
If we wanted to just change the existing token factory, what contracts would that affect? Just the EMP creator? If we want to reaudit and redeploy the EMP as a part of the perp work, could we make this change without affecting anything on the DVM side?

Reply:
Yes this would only affect the EMP contracts.

Reply:
I think we should consider just changing the implementation rather than copying it all over if we're going to add the token decimals change to the EMP as well. It seems like we're going through a lot of trouble to make this exclusive minter change in one contract but not the other.